docs/content.zh/docs/concepts/glossary.md

```diff
@@ -27,15 +27,13 @@ under the License.
 
 #### Flink Application Cluster
```

Comment:
link tag ,here.

Reply:
<a name="flink-application-cluster"></a>

Reply:
@RocMarshal , Thank you very much for your  review.